src/ui/pages/work/Workorder.js

```diff
@@ -618,6 +619,37 @@ class Workorder extends Entity {
         this.checklists && this.checklists.readActivities(this.state.workorder.number);
     };
 
+    saveHandler() {
+        // Validate all children and continue when all have passed
+        if (!this.validateFields(this.children)) {
+            this.props.showError('Several errors have occurred')
+            return
+        }
+
+        const terminalStatuses = ['T', 'TF', 'TT', 'TP', 'TI', 'TX'];
```

Comment:
Does this mean I cannot cancel a work order without completing the checklist?

Reply:
Proper way would be to check the system code instead, but this field is not available on the response.